src/nuxeo-cloud/additional-modules.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Additional Modules
+review:
+    comment: ''
+    date: '03-19-2020'
+    status: ok
+labels:
+    - nuxeo-cloud
+tree_item_index: 500
+toc: true
+---
+
+## Additional Environments
+
+Nuxeo Cloud customers can choose to purchase additional environments if required. These additional environments can be used to support training, additional testing efforts or support for internal teams.  
```

Comment:
change to "Nuxeo Cloud customers can choose to purchase additional environments. These additional environments can be used to support additional training or testing efforts."